axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/AxonServerEventScheduler.java

```diff
@@ -296,6 +302,18 @@ public Builder connectionManager(AxonServerConnectionManager axonServerConnectio
             return this;
         }
 
+        /**
+         * Sets the default context for this event scheduler to connect to.
+         *
+         * @param defaultContext for this scheduler to connect to.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        public Builder defaultContext(String defaultContext) {
```

Comment:
Naming this `defaultContext` feels off to me. There is no other way to publish to another context besides this one. I would expect it to just be `context`. Not setting it defaults it to the default of the `AxonServerConnectionManager`. 

As such I'm not sure whether the null-check is warranted as well. It is valid to be null.

Reply:
There is another way, if default context is skipped, connection manager is used to determine context which will use axon server properties or default to "default" context, which is backwards compatible behaviour. Besides that all other components, command bus, query bus has method with same name already...

Reply:
validation check is just in case you want to set default context, which protects user from making a mistake.
Its true if you set null it will delegate to connection manager, but probably not what users wants to do or how API should work... skipping calling defaultContext on builder is right way to do so, also backwards compatible

Reply:
but I understand your concert, and would change it if other components dont have same method already